### PR TITLE
Added CoinSpot and Salad

### PR DESCRIPTION
**Pull request by dimostheenis**

### CODE REVIEW

File: _data/sites.json [hunk 1]
```diff
@@ -4434,6 +4434,16 @@
         ]
     },
 
+        {
+        "name": "CoinSpot",
+        "url": "https://coinspot.zendesk.com/hc/en-us/articles/360001328796-Closing-your-Account",
+        "difficulty": "hard",
+        "notes": "Once you have removed all wallet balances, you need to create a ticket. ",
+        "domains": [
+            "https://www.coinspot.com.au/"
```
Comment (tupaschoal) [inline _data/sites.json:4443]:
```suggestion
            "coinspot.com.au"
```

File: _data/sites.json [hunk 2]
```diff
@@ -18157,6 +18167,17 @@
             "banaozel.sahibinden.com"
         ]
     },
+    
+        {
+        "name": "Salad",
```
Comment (tupaschoal) [inline _data/sites.json:18172]:
```suggestion

    {
        "name": "Salad",
```

File: _data/sites.json [hunk 3]
```diff
@@ -18158,6 +18168,17 @@
         ]
     },
 
+    {
+        "name": "Salad",
+        "url": "https://support.salad.com/article/45-how-can-i-delete-my-account",
+        "difficulty": "hard",
+        "notes": "You need to create a ticket."
```
Comment (tupaschoal) [inline _data/sites.json:18175]:
```suggestion
        "notes": "You need to create a ticket.",
```

Comment (dimostheenis) [reply _data/sites.json]: thank you

File: _data/sites.json [hunk 4]
```diff
@@ -4434,6 +4434,16 @@
         ]
     },
 
+        {
+        "name": "CoinSpot",
```
Comment (tupaschoal) [inline _data/sites.json:4438]:
```suggestion
    {
        "name": "CoinSpot",
```

**Review by tupaschoal — APPROVED** [review]

**Comment by tupaschoal** [comment]
Thanks for your contribution!

**Pull request merged**